src/middlewared/middlewared/plugins/boot_/boot_loader.py

```diff
@@ -0,0 +1,48 @@
+import os
+import platform
+import shutil
+import tempfile
+
+from middlewared.service import private, Service
+from middlewared.utils import run
+
+IS_LINUX = platform.system().lower() == 'linux'
+
+
```

Comment:
In this case seems to make more sense to have one file for linux and for freebsd? There is no shared code at all, just big `if` blocks.

Reply:
I am not sure i follow ? Are you saying we should have 2 files because there is no shared code ?

Reply:
yes